client/src/components/screens/PatientVariant/index.jsx

```diff
@@ -21,21 +22,24 @@ import { patientShape } from '../../../reducers/patient';
 import { variantShape } from '../../../reducers/variant';
 
 import Statement from '../../Query/Statement';
-import { fetchSchema, selectQuery, replaceQuery, replaceQueries, removeQuery, duplicateQuery, sortStatement, searchVariants, commitHistory, undo } from '../../../actions/variant';
+import { createNewQuery, fetchSchema, selectQuery, replaceQuery, replaceQueries, removeQuery, duplicateQuery, sortStatement, searchVariants, undo, addInstruction, removeInstruction, replaceInstruction, changeQueryTitle } from '../../../actions/variant';
 
 
 class PatientVariantScreen extends React.Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.handleCreateNewQuery = this.handleCreateNewQuery.bind(this);
     this.handleQuerySelection = this.handleQuerySelection.bind(this);
     this.handleQueryChange = this.handleQueryChange.bind(this);
     this.handleQueriesChange = this.handleQueriesChange.bind(this);
     this.handleQueriesRemoval = this.handleQueriesRemoval.bind(this);
     this.handleQueryDuplication = this.handleQueryDuplication.bind(this);
     this.handleStatementSort = this.handleStatementSort.bind(this);
-    this.handleCommitHistory = this.handleCommitHistory.bind(this);
```

Comment:
I don't understand, I thought we did this in https://github.com/cr-ste-justine/clin-frontend/pull/79/files but now we are undoing?

Reply:
We are now saving the history in the reducer.